docs/src/containers/home/info-tiles.svelte

```diff
@@ -12,16 +12,19 @@
     icon={GridIcon}
     title="{totalComponents} component{s(totalComponents)}"
     subtitle="and more to come!"
+    href="./docs/components/button"
```

Comment:
Maybe it is worth creating an `index.svx`, even if with something as simple as "check the components on the left"

Reply:
Nah, for now, the button serves better. Later we will create a proper page